pinot-integration-tests/src/test/java/com/linkedin/pinot/integration/tests/BaseClusterIntegrationTestSet.java

```diff
@@ -40,6 +40,7 @@
  * Shared set of common tests for cluster integration tests.
  * <p>To enable the test, override it and add @Test annotation.
  */
+@SuppressWarnings("unused")
```

Comment:
We keep some tests that covered by other tests, but if necessary, we can enable the unused tests for debugging purpose.